### DISCUSSION TITLE
Unable to import Plex users

### DISCUSSION THREAD

**Discussion by fpsware** [sct/overseerr#3331] [General]
The Plex server is setup properly, and Overseerr is communicating with my Plex server. 
![image](https://user-images.githubusercontent.com/51523764/218231972-fc5fdf8c-7199-4ba4-89bd-fe02abe443b0.png)

I'm unable to import Plex users. 
![image](https://user-images.githubusercontent.com/51523764/218231923-297160f0-8d7e-4a87-a5a5-52cb17ef18d0.png)

**Comment by AlFontal** [comment]
I have the same issue, even with the current develop branch installed. Any clues?

**Comment by mduijm** [comment]
Better to use this for support: https://discord.com/invite/overseerr
